gobblin-service/src/main/java/org/apache/gobblin/service/monitoring/GaaSJobObservabilityEventProducer.java

```diff
@@ -81,12 +85,14 @@ public abstract class GaaSJobObservabilityEventProducer implements Closeable {
   protected ObservableLongMeasurement jobStatusMetric;
   protected MultiContextIssueRepository issueRepository;
   protected boolean instrumentationEnabled;
+  protected boolean emitFlowObservabilityEvent;
   ContextAwareMeter getIssuesFailedMeter;
 
   public GaaSJobObservabilityEventProducer(State state, MultiContextIssueRepository issueRepository, boolean instrumentationEnabled) {
     this.state = state;
     this.issueRepository = issueRepository;
     this.instrumentationEnabled = instrumentationEnabled;
+    this.emitFlowObservabilityEvent = this.state.getPropAsBoolean(EMIT_FLOW_OBSERVABILITY_EVENT, false);
```

Comment:
do we not want these emitted by default? are job level events always emitted

Reply:
Yes its for backwards compatibility since every job status producer would need to implement the support for emitting flow level events